src/modules/dashboard/DashboardTopPanel.tsx

```diff
@@ -237,6 +245,23 @@ export const DashboardTopPanel = () => {
               >
                 <Trans>Claim</Trans>
               </Button>
+              {connected && (
+                <AddTokenDropdown
+                  poolReserve={{
+                    ...reserves[1],
+                    symbol: 'OG Points',
+                    underlyingAsset: '0x5607718c64334eb5174CB2226af891a6ED82c7C6',
```

Comment:
Seems valid for a new story in the backlog, don't think it's realistic to grok how rewards get implemented in a lifi widget integration PR because the hardcoded version is an hour ask turnaround for user confusion using hardcoding

Reply:
RIP - guess it's as easy as using `selectedReward` 💀

Reply:
Removed from PR